heureka/ui/src/hooks/useQueryClientFn.js

```diff
@@ -40,6 +41,14 @@ const useQueryClientFn = () => {
       },
     })
 
+    queryClient.setQueryDefaults(["components"], {
+      queryFn: async ({ queryKey }) => {
+        const [_key, options] = queryKey
+        console.log("useQueryClientFn::: queryKey: ", queryKey)
```

Comment:
it looks like the `console.log` is still there. Do you need it?

Reply:
yes, for now I keep all console.log for useQueryClientFn hook, but I removed the other console.log which is not needed anymore.